build.gradle

```diff
@@ -66,7 +66,7 @@ dependencies {
         implementation group: 'com.google.guava', name: 'guava', version: '33.0.0-jre'
```

Comment:
Do you know if this constraint is still needed too?  I have a feeling it isn't.

Reply:
You're probably right, let's see